roles/core/repositories_client/readme.rst

```diff
@@ -129,6 +130,7 @@ sources.list file.
 Changelog
 ^^^^^^^^^
 
+* 1.0.8: Add state parameter. Bruno Travouillon <devel@travouillon.fr>
```

Comment:
Could you also increment variable repositories_client_role_version in vars ?

Reply:
🤦‍♂️sure.